scanners/zap/zap.py

```diff
@@ -618,6 +619,29 @@ def _setup_passive_wait(self):
         }
         self.automation_config["jobs"].append(waitfor)
 
+    def _setup_replacer(self):
+        """Adds the replacer to the job list"""
+
+        if self.my_conf("replacer", default=False) is False:
```

Comment:
Since replacer is a dict, empty dict or None might be better defaults than a boolean.